Readme.md

```diff
@@ -43,6 +44,50 @@ Request mocking.
 |  ChatGPT integration  |               |
 |    Share requests     |               |
 
+
+## 🛠 Tech Stack
```

Comment:
Should do it as collapsible? Since README would get too bloated and long.

Reply:
I agree that this would bee long.
Maybe move this to the end.

Reply:
We can add this as separate stack.md and refer to its link readme.md

Reply:
@shravan20 It's a great idea, I've committed the changes.

Reply:
@senderh55 Saw it only now. Thank you sender merging this PR.